test/Unipool.js

```diff
@@ -181,28 +186,30 @@ contract('Unipool', function ([_, wallet1, wallet2, wallet3, wallet4]) {
         });
 
         it('One staker on 2 durations with gap', async function () {
-            // 72000 SNX per week for 1 weeks
-            await this.pool.notifyRewardAmount(web3.utils.toWei('72000'), { from: wallet1 });
+            // 72000 ANT for 60 days
+            await this.ant.approveAndCall(this.pool.address, web3.utils.toWei('72000'), ZERO_DATA, { from: wallet1 });
+            expect(await this.ant.balanceOf(this.pool.address)).to.be.bignumber.almostEqualDiv1e18(web3.utils.toWei('72000'));
 
             await this.pool.stake(web3.utils.toWei('1'), { from: wallet1 });
 
-            await timeIncreaseTo(this.started.add(time.duration.weeks(2)));
+            await timeIncreaseTo(this.started.add(time.duration.days(60)));
 
             expect(await this.pool.rewardPerToken()).to.be.bignumber.almostEqualDiv1e18(web3.utils.toWei('72000'));
             expect(await this.pool.earned(wallet1)).to.be.bignumber.almostEqualDiv1e18(web3.utils.toWei('72000'));
 
-            // 72000 SNX per week for 1 weeks
-            await this.pool.notifyRewardAmount(web3.utils.toWei('72000'), { from: wallet1 });
+            // 72000 ANT for 90 days
+            await this.ant.approveAndCall(this.pool.address, web3.utils.toWei('72000'), ZERO_DATA, { from: wallet1 });
 
-            await timeIncreaseTo(this.started.add(time.duration.weeks(3)));
+            await timeIncreaseTo(this.started.add(time.duration.days(90)));
 
             expect(await this.pool.rewardPerToken()).to.be.bignumber.almostEqualDiv1e18(web3.utils.toWei('144000'));
             expect(await this.pool.earned(wallet1)).to.be.bignumber.almostEqualDiv1e18(web3.utils.toWei('144000'));
         });
 
-        it('Notify Reward Amount from mocked distribution to 10,000', async function () {
-            // 10000 SNX per week for 1 weeks
-            await this.pool.notifyRewardAmount(web3.utils.toWei('10000'), { from: wallet1 });
+        it('Reward from mocked distribution to 10,000', async function () {
```

Comment:
This test is failing for me on line 224. It is weird, I haven't looked deep into it but it may worth debugging it

Reply:
What was the error? It works fine for me.

Reply:
It's passing now, must be flaky then 🤷

Reply:
I think I found this issue, see: https://github.com/aragonone/Unipool/pull/7